staging/src/k8s.io/apiserver/pkg/storage/interfaces.go

```diff
@@ -302,4 +302,9 @@ type ListOptions struct {
 	// event containing a ResourceVersion after which the server
 	// continues streaming events.
 	SendInitialEvents *bool
+	// SkipCache bypasses the server cache, forcing the request to be served directly from etcd.
+	//
+	// Caution: This dramatically increases the request resource cost.
+	// Use only for debugging cache staleness or corruption. Not intended for regular client usage.
+	SkipCache bool
```

Comment:
Currently this just doesn't work - you need to explicitly set this option based on input here:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/registry/generic/registry/store.go#L366

Reply:
Which suggests we probably need some integration test to validate this work too.

Reply:
Right, was looking to setup a local cluster to test it. 

I also thought about integration tests, but I was not sure how to confirm that request was served by etcd.

Reply:
Maybe via some metrics?

Reply:
Ok, it was simple. We already have a ` TestListOptions` that validates use of watch cache.